dev/bots/allowlist.dart

```diff
@@ -25,7 +25,6 @@ const Set<String> kCorePackageAllowList = <String>{
   'boolean_selector',
   'characters',
   'charcode',
-  'cli_util',
```

Comment:
This should be done in its own PR https://github.com/flutter/flutter/pull/99037 as it's totally unrelated to your docstring change.
Please hold off merging this until https://github.com/flutter/flutter/pull/99037 lands (it should merge cleanly so you don't need to update this PR).

Reply:
I'd still suggest rebasing this PR to the current master to make this diff go away. It will look odd for future archeologists looking at this PR.

Reply:
https://github.com/flutter/flutter/pull/99037 has been submitted, so this PR should be good to go.